src/test/synthetic_table_generator_test.cpp

```diff
@@ -71,13 +71,10 @@ TEST_P(SyntheticTableGeneratorDataTypeTests, IntegerTable) {
   std::transform(all_segment_encoding_specs.begin(), all_segment_encoding_specs.end(),
                  std::back_inserter(supported_segment_encodings), replace_unsupporting_encoding_types);
 
-  const auto test_data_types = std::vector<DataType>(supported_segment_encodings.size(), tested_data_type);
-  const auto test_data_distributions =
-      std::vector<ColumnDataDistribution>(supported_segment_encodings.size(), std::get<1>(GetParam()));
-  const auto column_names = std::vector<std::string>(supported_segment_encodings.size(), "column_name");
+  ColumnSpecification column_specification = {std::get<1>(GetParam()), tested_data_type, "column_name", std::nullopt};
```

Comment:
The last std::nullopt could be omitted here

Reply:
Resolved in 9425a461a6a5665302b3e1bf26d383b825b0ffc8.